addons/sys_io/fnc_ts3ChannelCheck.sqf

```diff
@@ -0,0 +1,25 @@
+/*
+ * Author: Tim Beswick
+ * Checks if main display is visible and sets server name, triggering ts3 channel move.
+ *
+ * Arguments:
+ * None
+ *
+ * Return Value:
+ * None
+ *
+ * Example:
+ * [] call acre_sys_server_fnc_teamspeakChannelCheck
+ *
+ * Public: No
+ */
+#include "script_component.hpp"
+
+[{
+    params ["", "_idPFH"];
+
+    if (GVAR(serverStarted) && {!(isNull (findDisplay 46))}) exitWith {
```

Comment:
This ensures it only moves people once the briefing is complete. On the server when you press OK in the lobby, you come to the briefing screen, then there is the Continue button which starts the game. Personally I think being able to chat normally in the briefing screen is better than being moved at that stage.